src/MoBi.Presentation/Presenter/SimulationChartPresenter.cs

```diff
@@ -40,9 +37,7 @@ protected override void MarkChartOwnerAsChanged()
 
       protected override string CurveNameDefinition(DataColumn column)
       {
-         var simulationForDataColumn = _simulations[column.Repository];
-
-         return _quantityDisplayPathMapper.DisplayPathAsStringFor(simulationForDataColumn, column);
+         return _curveNamer.CurveNameForColumn(_simulations[column.Repository], column, addSimulationName: false);
```

Comment:
Let's create another issue for that so that we do not forget. I'd like to avoid doing to many changes in the Curve department at the moment